homeassistant/components/spc/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for SPC integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from aiohttp import ClientError
+from pyspcwebgw import SpcWebGateway
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.helpers import aiohttp_client
+
+from . import CONF_API_URL, CONF_WS_URL, DOMAIN
+
+
+class SpcConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for SPC."""
+
+    VERSION = 1
+
+    async def async_step_import(
+        self, import_config: dict[str, Any]
+    ) -> ConfigFlowResult:
+        """Import a config entry from configuration.yaml."""
+        return await self.async_step_user(import_config)
```

Comment:
The thing with the import flow is that we should either create the entry for the user, or abort. So we should make sure we don't return with errors to the user

Reply:
I _think_ I've addressed this but I am not certain.

Reply:
I'll have a look, be sure to check an integration like `nhc` who recently got a config flow import

Reply:
Cool! Will check it out! thanks for the tip.